src/Components/FirestoreList.jsx

```diff
@@ -1,32 +1,21 @@
 import React from 'react';
-import { FirestoreDocument } from 'react-firestore';
+import { FirestoreCollection } from 'react-firestore';
 import List from './List/List';
 
 /**
  * Retrieves token from local storage and accesses the saved items list from Firestore
  */
 
 export default function FirestoreList({ token }) {
-  // Extracts item objects from Firestore into an array
-  const dataToArray = (data) => {
-    const itemList = [];
-    const storedList = Object.entries(data);
-    storedList.forEach(([key, value]) => {
-      if (key !== 'id') {
-        itemList.push(value);
-      }
-    });
-    return itemList;
-  };
-
   return (
-    <FirestoreDocument
-      path={`lists/${token}`}
+    <FirestoreCollection
+      path={`lists/${token}/items`}
+      // sort="lastPurchased:desc"
```

Comment:
It's too bad that we couldn't get sort to work with multiple fields. Googling also didn't give me many results since the `react-firestore` package is outdated. 😅